x-pack/test/functional/apps/security/field_level_security.js

```diff
@@ -14,11 +14,14 @@ export default function ({ getService, getPageObjects }) {
   const retry = getService('retry');
   const log = getService('log');
   const PageObjects = getPageObjects(['security', 'settings', 'common', 'discover', 'header']);
+  const kibanaServer = getService('kibanaServer');
 
   describe('field_level_security', () => {
     before('initialize tests', async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/security/flstest/data'); //( data)
-      await esArchiver.load('x-pack/test/functional/es_archives/security/flstest/kibana'); //(savedobject)
```

Comment:
It looks like this was the only reference to the `x-pack/test/functional/es_archives/security/flstest/kibana` directory, can we delete it now?

Reply:
ah yup! I forgot to cherry pick the deletion.  On it!